frontend/src/cards/MapCard.tsx

```diff
@@ -1,33 +1,34 @@
-import React, { useState } from "react";
-import Alert from "@material-ui/lab/Alert";
+import { CardContent, Grid } from "@material-ui/core";
```

Comment:
Can you sync with Krista re: the linting issues & subsequent merge conflicts that have been going on?

Reply:
Yeah ill sync with her